src/AWS.Deploy.Recipes.CDK.Common/AWS.Deploy.Recipes.CDK.Common.csproj

```diff
@@ -24,6 +25,7 @@
 
   <ItemGroup>
     <None Include="..\..\icon.png" Pack="true" PackagePath="" />
+    <None Include="..\..\README.md" Pack="true" PackagePath="" />
```

Comment:
Can we put in a very small separate README file for `AWS.Deploy.Recipes.CDK.Common` that just says this a packages used for .NET deployment project created by `AWS.Deploy.Tool`?  I think reusing the README for AWS.Deploy.Tool will be confusing.

Reply:
Done